src/Simplesoft.Database.Contracts/ITransaction.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SimpleSoft.Database
+{
+    /// <summary>
+    /// Creates and manages the current transaction
+    /// </summary>
+    public interface ITransaction : IDisposable
```

Comment:
This interface must extend `IAsyncDisposable` when compiling for NET Standard 2.1